mobile/src/screens/meditation-home/libs/constants.ts

```diff
@@ -0,0 +1,6 @@
+const mockedData = [
+  { id: 1, title: 'Meditation' },
+  { id: 2, title: 'Relaxing sounds' },
+  { id: 3, title: 'Breath exercise' },
+];
+export { mockedData };
```

Comment:
I think there's no need to name this array 'mocked' anymore because these are the actual data

Reply:
@danilittaa please, show me actual data for meditation topics 😊 i know there are actual data for entries , but i havent seen topics

Reply:
@danilittaa okay, i can call it navigationItems, but QA asked to make these three topics here, thats why i did it like that. i think it needs more investigation